turbo/snapshotsync/external_downloader_grpc.pb.go

```diff
@@ -4,6 +4,7 @@ package snapshotsync
 
```

Comment:
You can delete this, it will now need to live in gointerfaces package

Reply:
I'll do it in a separate PR, it you don't mind.